Portfolio/src/App.vue

```diff
@@ -3,96 +3,95 @@ import { ref } from 'vue';
 import Copyright from "../src/components/Copyright.vue"
 import { homeViewContent } from './views/home-view-content';
+import Wrapper from "@/Wrapper.vue";
 
-const isMenuShown = ref(false)
 
+const isMenuShown = ref(false)
 
 </script>
 
 <template>
-    <header>
-        <div class="router-wrapper">
-            <button @click="isMenuShown = !isMenuShown" class="menuButton">
-                <div class="line"></div>
-                <div class="line"></div>
-                <div class="line"></div>
-            </button>
-
-            <nav>
-                <ul class="navbar">
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e; ;" to="/">Home
-                        </RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e; " to="/work">Work</RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/about">About</RouterLink>
-                    </li>
-                    <li>
-                        <RouterLink style="text-decoration: none; color: #1e1e1e;" to="/contact">Contact</RouterLink>
-                    </li>
-                </ul>
-            </nav>
-
-        </div>
-    </header>
-
-    <div v-if="isMenuShown" class="openedMenu" style="font-family: 'RalewayBold'; text-transform: uppercase;">
-        <ul class="menu-links">
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; ;" to="/">Home
-                </RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff; " to="/work">Work</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/about">About</RouterLink>
-            </li>
-            <li>
-                <RouterLink style="text-decoration: none; color: #ffff;" to="/contact">Contact</RouterLink>
-            </li>
-        </ul>
-
-        <div class="menu-fullName">
-            <h1 class="menu-firstName">{{ homeViewContent.firstName }}</h1>
-            <h1 class="menu-lastName">{{ homeViewContent.lastName }}</h1>
+    <Wrapper>
+        <header>
+            <div class="router-wrapper">
+                <button @click="isMenuShown = !isMenuShown" class="menuButton">
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }">
+                    </div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                    <div class="line" :class="{ 'whiteLine': isMenuShown }"></div>
+                </button>
+
+                <nav>
+                    <ul class="navbar">
+                        <li>
+                            <RouterLink style="text-decoration: none; color: #1e1e1e; ;" to="/">Home
```

Comment:
This `style="text-decoration: none; color: #1e1e1e; ;"` is called "inline-styles" and they're generally frowned upon.
There is a few reasons why this is bad practice, on of them being:

_Inline styles add extra weight to your HTML file, which increases the amount of data that your browser has to download and parse. This can slow down your page rendering and user experience, especially on mobile devices or slow connections._

Reply:
Done